src/Magnum/Math/Geometry/Distance.h

```diff
@@ -162,6 +162,34 @@ class Distance {
          * the square root.
          */
         template<class T> static T lineSegmentPointSquared(const Vector3<T>& a, const Vector3<T>& b, const Vector3<T>& point);
+
+        /**
+         * @brief Distance of point from plane, scaled by the length of the planes normal.
+         * @param point the point
+         * @param plane the plane
+         * @returns a value `> 0` if point is on positive half space,
+         *                  `< 0` if point is on negative half space,
+         *                  `0` if point is on the plane.
```

Comment:
Well, it is a plain text paragraph (not a list in case that is what you meant)? It's just easier to read in code this way, I find.